<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winStatus</class>
 <widget class="QWidget" name="winStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lHead">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Latin Modern Roman Caps</family>
     <pointsize>33</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Aktuel status</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bAbort">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>461</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tilbage</string>
   </property>
  </widget>
  <widget class="QLabel" name="lTemp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Seneste temperatur:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lHumi">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Seneste Fugtighed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lMove">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bevægelse:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lSprink">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sprinkler aktivitet:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tTemp">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>40 °C</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tHumi">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>20 %</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tMove">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingen</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tSprink">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Tændt</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
